tests/test_shipments.py

```diff
@@ -339,6 +359,15 @@ def test_create_with_location(self):
                                                           'storage_credentials_id': STORAGE_CRED_ID
                                                           })
 
+        one_location_profiles_disabled, content_type = create_form_content({'ship_from_location.name': LOCATION_NAME,
```

Comment:
`one_location_profiles_disabled` isn't used anywhere. We definitely want a test case that checks that shipment creation works when profiles is disabled. Specifically, for this PR, we need to make sure that the storage credentials permissions check doesn't happen when profiles is disabled.